ext/js/data/sandbox/anki-note-data-creator.js

```diff
@@ -353,6 +385,7 @@ export class AnkiNoteDataCreator {
         const expressions = this.createCachedValue(this._getTermExpressions.bind(this, dictionaryEntry));
         const frequencies = this.createCachedValue(this._getTermFrequencies.bind(this, dictionaryEntry));
         const pitches = this.createCachedValue(this._getTermPitches.bind(this, dictionaryEntry));
+        const transcriptions = this.createCachedValue(this._getTermPhoneticTranscriptions.bind(this, dictionaryEntry));
```

Comment:
This is named just `transcriptions` while `phoneticTranscriptions` is used above. Should `phoneticTranscriptions` be used for both?